django/db/backends/base/base.py

```diff
@@ -286,15 +307,35 @@ def get_autocommit(self):
         self.ensure_connection()
         return self.autocommit
 
-    def set_autocommit(self, autocommit):
+    def set_autocommit(self, autocommit, force_begin_transaction_with_broken_autocommit=False):
```

Comment:
It's explained in a comment within the body of the method, but having an explanation in the docstring (also or instead) would be good.

Neither myself nor @aaugustin nor @pelme could come up with a shorter name that seemed accurate, but ideas are welcome.